pkg/kritis/admission/admission_test.go

```diff
@@ -115,14 +115,17 @@ func Test_AdmissionResponse(t *testing.T) {
 	}
 	for _, tc := range tcs {
 		t.Run(tc.name, func(t *testing.T) {
-			mReviewer := func(client metadata.Fetcher) reviewer {
+			mReviewer := func() reviewer {
 				return testutil.NewReviewer(tc.reviewGAPErr, tc.reviewISPErr, tc.expectedMsg)
 			}
 			mockConfig := config{
 				retrievePod: mockValidPod(),
-				fetchMetadataClient: func(config *Config) (metadata.Fetcher, error) {
+				fetchMetadataClient: func(config *Config) (metadata.ReadWriteClient, error) {
 					return testutil.NilFetcher()()
```

Comment:
nit: could we rename this to `NilClient` or `NilReadWriteClient` to be consistent with the new naming?